tests/unit/snapshots/banksnapshots.test.ts.snap

```diff
@@ -17622,6 +17620,45 @@ exports[`OSB Creatables 1`] = `
     },
     "type": "unpack",
   },
```

Comment:
Packs should be added to cl so they are shown under /cl creatables. Unpacking sets shouldn't be added to cl since that would affect the source cl of the item.